packages/kbn-apm-types/src/es_fields/apm.ts

```diff
@@ -194,7 +240,68 @@ export const CLIENT_GEO_REGION_ISO_CODE = 'client.geo.region_iso_code';
 export const CLIENT_GEO_COUNTRY_NAME = 'client.geo.country_name';
 export const CLIENT_GEO_CITY_NAME = 'client.geo.city_name';
 export const CLIENT_GEO_REGION_NAME = 'client.geo.region_name';
+export const CLIENT_IP = 'client.ip';
 
 export const CHILD_ID = 'child.id';
 
 export const LOG_LEVEL = 'log.level';
+
```

Comment:
The Otel-specific fields (under `attributes` and `resource.attributes` should not be part of the new data mapping - we use those fields only in the metadata table (we need to confirm if we show them anywhere else). We should keep the mappers close to the APM types 

cc: @AlexanderWert and @felixbarny I remember we discussed that, let me know if I am missing something here

Reply:
I was expecting the mapping in `es_fields_mappings.ts` to not produce `Record<string, unknown[]>` but instead types such as `TransactionRaw`